### PR TITLE
Notes for LCWG Call 58 - Aug 24th 2023

### LINKED ISSUE #58: LCWG Call 58 - Aug 24th 2023
LCWG Call 58 - Aug 24th 2023

### PR DESCRIPTION
**Pull request by pacu**
[#58] Notes for LCWG Call 58 - Aug 24th 2023
    
closes #58

### CODE REVIEW

File: meeting-notes/lcwg-dev-51.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Light Client Working Group Devs Meeting 51
+### Meeting Date/Time: Thursday, June 1st, 2023 17:00 UTC
+### Meeting Duration: 45 minutes
+### [Video of the meeting](not-recorded)
+### Moderator: Nick ECC
+### Attendees: Dan (ZF), Mandeep (Nighthawk), John (ZF), Matt (Nighthawk), Adi (Nighthawk), Kriss (ECC)
```
Comment (nuttycom) [inline meeting-notes/lcwg-dev-51.md:6]:
```suggestion
### Attendees: Dan (ZF), Mandeep (Nighthawk), John (ZF), Matt (Nighthawk), Adi (Nighthawk), Kris (ECC)
```

File: meeting-notes/lcwg-dev-58.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+#### Darksidewalletd Testing and Sbs 
+Kris: There are Darksidetests that need review and fixing on the SDKs on [This PR](https://github.com/zcash/ZcashLightClientKit/pull/1191). Pacu will review and prioritize any work if possible since there are other tasks in his backlog.
+
+Kris also suggested that the Dlwd tests have a definition language so that tests can be written once and ran on whichever wallet and those tests suites can be user on z.cash to actually get it listed on the official wallet list on the website.
+
+https://github.com/zcash/ZcashLightClientKit/pull/1191
+
+
+###  Action Items from previous meeting
+- ~Enable ZIP-317 booleans on the sdk and do small releases~. This has been deprioritized by ECC to focus on Sbs. Minimum fee of 10000 zat may or may not be included in next release.
```
Comment (nuttycom) [inline meeting-notes/lcwg-dev-58.md:56]: The ECC team discussed this after the call and verified that the next version of each of the SDKs will ship with a 10,000 ZAT fixed fee.

File: meeting-notes/lcwg-dev-57.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- Str4d says the problem might be related to transaction expiry heights not being set properly or
+ZIP-317 not being enforced properly. Recommended expiry height is 40 blocks since Blossom. If a transaction is stuck
+- Reports on graphana show that the mempool started to grow up in size within the previous 15 days. One workaround would be shrinking the mempool size on nodes, setting expiry heights and enforcing ZIP-317 fees.
+- According to NH team, the mempool is being fully populated with small transactions with small amount of actions and the blocks are barely including any of them, therefore causing user txs
+being unmined and expired.
+
+Pacu: What actions that can be taken to avoid workarounds and get the concrete solutions out?
+
+Kris: 
+- Review and get void out for the changes of transaction proposal here: https://github.com/zcash/librustzcash/pull/891
```
Comment (nuttycom) [inline meeting-notes/lcwg-dev-57.md:20]: "void" ?

File: meeting-notes/lcwg-dev-58.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+### ZWCD Updates
+Pacu is working in darksidewalletd benchmarks for Zingo that will be extended and made available to other wallets. Also reviewing PRs from NH and ECC. 
+
+He requested suggestions on improving regtest performance because the testcase generation was taking too much time and had to move on to a different approach.
+
+For this Kris pointed out [this code](https://github.com/zcash/zcash/blob/master/qa/rpc-tests/test_framework/util.py) that could help set up cached regtest states and then use the resulting wallet directory to generate the darksidewalletd datasets.
```
Comment (nuttycom) [inline meeting-notes/lcwg-dev-58.md:28]: A better example: https://github.com/zcash/zcash/commit/ff02805e99bfa2ba9498f26eb49a791f6bb2f36f#diff-4aa4f998e880e58d5a14ed12c81eb7664478385b4ea185cab9dec0cd84044d63R40-R143

File: meeting-notes/lcwg-dev-57.md (hunk 3 above)
Comment (pacu) [reply meeting-notes/lcwg-dev-57.md:20]: LOL apparently VS Code thought it could autocorrect

File: meeting-notes/lcwg-dev-58.md (hunk 2 above)
Comment (pacu) [reply meeting-notes/lcwg-dev-58.md:56]: Thanks for clarifying this! I'll will include in the notes as a follow up

File: meeting-notes/lcwg-dev-58.md (hunk 4 above)
Comment (pacu) [reply meeting-notes/lcwg-dev-58.md:28]: added!

**Review by nuttycom — APPROVED** [review]
Looks good, thanks for doing this, Pacu!

**Comment by pacu** [comment]
please merge if so, I don't have write access so I'm not allowed to do it myself

**Pull request merged**